include/crow/http_server.h

```diff
@@ -175,21 +177,27 @@ namespace crow
         }
 
     private:
-        asio::io_service& pick_io_service()
+        int pick_io_service_idx()
         {
-            // TODO load balancing
-            roundrobin_index_++;
-            if (roundrobin_index_ >= io_service_pool_.size())
-                roundrobin_index_ = 0;
-            return *io_service_pool_[roundrobin_index_];
+            int min_queue_idx = 0;
+
+            // TODO improve load balancing
+            for (uint i = 1; i < task_queue_length_pool_.size(); i += 1)
```

Comment:
I'm confused as to why `i` starts at 1 rather than 0.

Also since `concurrency` is a `uint16_t`, it doesn't make sense to use a 32bit uint.

Lastly it might be best to use `i++`

Reply:
It starts with 1 because `min_queue_idx` is initialised at 0 so it wouldn't make sense to compare the index 0 to 0 on the first iteration.
Ok, I changed it to `uint16_t`.